tensorboard/webapp/app_module.ts

```diff
@@ -31,6 +32,7 @@ import {MatIconModule} from './mat_icon_module';
 @NgModule({
   declarations: [AppContainer],
   imports: [
+    AnalyticsModule,
```

Comment:
We are not going to do analytics on OSS. Does it even make sense to specify this line? Let's omit the dependencies we do not need.

Reply:
Removed.